README.md

```diff
@@ -42,21 +51,13 @@ To plot the output as a heatmap:
 
```

Comment:
In the above code, can we change to since `example_som_input_df` isn't actually defined?

```
    # Append results to the input data
    df['cluster'] = clusters
    # Find mean of each cluster
    df_mean = df.groupby(['cluster']).mean()
```